packages/daemon/src/pet-store.js

```diff
@@ -104,22 +105,59 @@ export const makePetStoreMaker = (filePowers, locator) => {
       const petNamePath = filePowers.joinPath(petNameDirectoryPath, petName);
       const petNameText = `${formulaIdentifier}\n`;
       await filePowers.writeFileText(petNamePath, petNameText);
-      changesTopic.publisher.next({ add: petName });
+      const formularIdentifierPair = parseFormulaIdentifier(formulaIdentifier);
+      changesTopic.publisher.next({
+        add: petName,
+        value: formularIdentifierPair,
+      });
     };
 
-    const list = () => harden([...petNames.keys()].sort());
+    /**
+     * @param {string} petName
+     * @returns {import('./types.js').FormulaIdentifierPair}
+     */
+    const formulaIdentifierPairForPetname = petName => {
```

Comment:
I’ve elsewhere used `Name` consistently to mean pet name in method names. I’m not against expanding these all to `PetName` but would want to do it in a holistic style change.

```suggestion
    const formulaIdentifierPairForName = petName => {
```